app/views/static_pages/contact.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide(:title, t(".Contact")) %>
+<h1><%= t ".Contact" %></h1>
+<p>
+  <%= t ".Contact page:" %>
```

Comment:
`.Contact page:` có biến có space thế này ah em? 

biến thì ko bắt đầu với chữ cái in hoa